modules/apps/commerce/commerce-checkout-web/src/main/resources/META-INF/resources/js/AddressSubtypeAutocomplete.tsx

```diff
@@ -0,0 +1,399 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import Autocomplete from '@clayui/autocomplete';
+import ClayButton from '@clayui/button';
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClayModal, {useModal} from '@clayui/modal';
+import classnames from 'classnames';
+
+// @ts-ignore
+
+import {CommerceServiceProvider} from 'commerce-frontend-js';
+import {openToast} from 'frontend-js-components-web';
+import React, {useEffect, useMemo, useState} from 'react';
+
+interface IAddressSubtypeAutocompleteInputProps {
+	onChange(value: string): void;
```

Comment:
Sort

Reply:
The SF required the function as first in the interface